package.json

```diff
@@ -7,7 +7,7 @@
   "repository": "https://github.com/gruntjs/grunt.git",
   "license": "MIT",
   "engines": {
-    "node": ">=8"
+    "node": ">=16"
```

Comment:
FYI: I think that if you're following Semver, this is a breaking change, requiring a `major` version bump. Yet this was published in a `minor` version bump (`1.5.X` => `1.6.0`).

This means that packages (like [`spectaql`](https://github.com/anvilco/spectaql/blob/main/package.json)) that had something like this in their dependencies will now break for installations on Node < 16:

```js
{
  ...
  "dependencies": {
    ...
    "grunt": "^1.5.3",
    ...
  }
  ...
}
```

@vladikoff @XhmikosR